src/style.css

```diff
@@ -0,0 +1,452 @@
+}
+
+.card_image {
+  width: 265px;
+  height: 230px;
+  border-radius: 5px;
+}
+
+.card_body {
+  width: 85%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px 0;
+}
+
+.card_title {
+  font-size: 1.2em;
+}
+
+.like_container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.like_text {
+  margin-top: 5px;
+  width: 60px;
+}
+
+.like {
+  cursor: pointer;
+}
+
+.like:hover {
+  transform: scale(1.1);
+}
+
+.buttons_container {
+  width: 85%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.btn {
+  color: #e7e7e7;
+  padding: 10px;
+  font-size: 1em;
+  font-weight: 500;
+  cursor: pointer;
+  border: none;
+  background-color: #1f1e1e;
+  border-radius: 20px;
+  box-shadow: rgba(0, 0, 0, 0.2) 0 12px 28px 0, rgba(0, 0, 0, 0.1) 0 2px 4px 0, rgba(255, 255, 255, 0.05) 0 0 0 1px inset;
+}
+
+.btn-close {
+  cursor: pointer;
+  margin-bottom: 20px;
+  border-radius: 10%;
+  display: flex;
+  background-color: #272727;
+  color: #e7e7e7;
+  box-shadow: rgba(0, 0, 0, 0.2) 0 12px 28px 0, rgba(0, 0, 0, 0.1) 0 2px 4px 0, rgba(255, 255, 255, 0.05) 0 0 0 1px inset;
+}
+
+.btn-close:hover {
+  background-color: #2e2c2c;
+}
+
+.btn:hover {
+  background-color: #2e2c2c;
+}
+
+.btn-outline-dark {
+  margin-left: 10px;
+}
+
+/* Footer */
+footer {
+  display: flex;
+  padding: 10px;
+  width: 100%;
+  background-color: #272727;
+  box-shadow: rgba(0, 0, 0, 0.2) 0 12px 28px 0, rgba(0, 0, 0, 0.1) 0 2px 4px 0, rgba(255, 255, 255, 0.05) 0 0 0 1px inset;
+}
+
+.footer_container {
+  margin: auto;
+  width: 90%;
+  height: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer_txt {
+  font-size: clamp(1rem, 2vw, 1.3rem);
+  height: 100%;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  text-align: center;
+}
+
+#reservation_modal {
+  color: #1f1e1e;
+  position: relative;
+  z-index: 1;
+  background-color: #fff;
+  margin: 6.5% auto;
+  padding: 1.5rem;
+  font-size: 1.1rem;
+  border-radius: 25px;
+  border: 1px solid #888;
+  width: 50%;
+  min-width: 370px;
+  max-width: 550px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  -webkit-animation-name: animatetop;
+  -webkit-animation-duration: 0.4s;
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+}
+
+@-webkit-keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+
+  to {
+    top: 1rem;
+    opacity: 1;
+  }
+}
+
+@keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+
+  to {
+    top: 1rem;
+    opacity: 1;
+  }
+}
+
+#modal-container {
+  display: none;
+  position: fixed;
+  left: 0;
+  z-index: 1;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.4);
+  backdrop-filter: blur(15px);
+  overflow: auto;
+}
+
+#reservation_modal span {
+  float: right;
+  background-color: #272727;
+  color: #e7e7e7;
+  font-size: 1.6rem;
+  clear: "none";
```

Comment:
- [ ] On the other hand, the linter is also complaining about this line, and it looks like is because `"none"` inside quotation marks is not a valid value for this property. According to [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/clear#syntax), it should be plain `none`. Please consider fixing this typo! 🤓